### ISSUE TITLE
Crash when opening a file

### ISSUE THREAD

**Issue by krashy-i386** [jlesage/docker-handbrake#254]
I've installed your container, but when i try to load a video it just crashes. 
I've tried some mp4s and webms with no luck.
Any idea why?

https://user-images.githubusercontent.com/61975168/223448772-b971cafc-1985-4abc-a906-de44bd00988c.mp4

Log file: [_handbrake-handbrake-1_logs.txt](https://github.com/jlesage/docker-handbrake/files/10910333/_handbrake-handbrake-1_logs.txt)

Also the permissions on the volumes folders should be ok (container running with pid and gid 1000 which is the same for the folders)

**Comment by Mellumen** [comment]
I have the same exact problem. The same log output. The automated encoding from /watch folder works for me, but not if I press and use the web GUI. Have you tried this?

**Comment by mavrik132** [comment]
I am also experiencing this restarting problem when opening a file through the UI.  It always seems to crash when generating the 9th of 10 preview images.  The last working version for me is v1.25.1.  Version v23.02.1 and newer seem to be affected.

[handbrake-logs.txt](https://github.com/jlesage/docker-handbrake/files/11186341/handbrake-logs.txt)

**Comment by banning4** [comment]
Same Issue for me with the same log file

**Comment by jlesage** [comment]
Can you check the log file located at `ghb/Activity.log*` under the directory you mapped to `/config`?
Do you see the crash with all video files ?  Or some are working ?

**Comment by Ysval** [comment]
Hello,

Same problem here :
proxmox/ubuntu/handbrake -> handbrake goes wild and crash on any opened source.
The only "logs" i could find relative to the crash is this : 
![image](https://github.com/jlesage/docker-handbrake/assets/133708622/36cef9f0-0ebd-49df-8dfb-20fb4aff3b1b)
No nginx error log, neither for hb.

Regards

**Comment by Ysval** [comment]
After some more testing :
There is a problem on QEMU/KVM (proxmox).
If the docker host processors type is set to any QEMU type, like : 
![image](https://github.com/jlesage/docker-handbrake/assets/133708622/c86cc5c9-7339-4e02-8355-90a32e65ba6b)
-> **_it will crash_**.

Resolved (in my case) by setting the "real" processor type (intel/amd/...).

Hope it might help others.

**Comment by Porklaser** [comment]
I was experiencing the same issue and @Ysval 's fix worked for me.

My container server is running in proxmox VM and was set to the default processor type of kvm64. Switching it to type "host" fixed the issue.

**Comment by mavrik132** [comment]
I also am running in a Proxmox VE VM. After changing the processor type from kvm64 to host, the problem is gone. 

Thanks everyone!

**Comment by AngryAnt** [comment]
Would ofc. still be preferable to not have a crash be expected behaviour ;)

**Comment by jlesage** [comment]
Are you running Docker in a VM ?

**Comment by Snake883** [comment]
> Are you running Docker in a VM ?

Via Unraid/Portainer.

**Comment by jlesage** [comment]
Is unRAID running in a VM ?

**Comment by zkhcohen** [comment]
> Are you running Docker in a VM ?

Any ideas on this issue? I am running Docker in a VM with "Default (kvm64)" set due to a complicated QSV issue.

I'm having the exact same symptoms as the people above.

Tdarr and Jellyfin are not having this issue when encoding using ffmpeg, suggesting that there's another issue with this particular container.

**Comment by AngryAnt** [comment]
> Any ideas on this issue?

Yes. A workaround for your concrete issue exists. A fix does not.

> a VM with "Default (kvm64)" set due to a complicated QSV issue

I'm sorry, but you just explained why this is happening - it has already been pointed out in this thread. Expected behaviour with this configuration is currently a hard crash on file open. The only known workaround at present is to not use the emulated processor types in your VM configuration.

**Comment by dongshimou** [comment]
same error. need change vm cpu ?
